apps/frontend/src/components/cards/SeverityChart.vue

```diff
@@ -57,12 +57,26 @@ export default class SeverityChart extends Vue {
     ];
   }
 
+  valueToSeverity(severity: string): Severity {
+    switch (severity) {
+      case 'low':
+        return 'low'
+      case 'medium':
+        return 'medium'
+      case 'high':
+        return 'high'
+      case 'critical':
+        return 'critical'
+    }
+    return 'none'
+  }
```

Comment:
This code seems rather redundant, I think it exists in at least 1 other place and the list of valid types exists in other places as well: https://github.com/mitre/heimdall2/blob/8c2a8b30c6161769dd2454d3e49a99d5de379141/apps/frontend/src/store/search.ts#L42

It would be great to deduplicate the list of possible severities into 1 location and then import it everywhere else, then have something like this in all of the locations this happens:

```
    const severityTypes: Severity[] = ['none', 'low', 'medium', 'high', 'critical'];
  valueToSeverity(severity: string): Severity {
    if(severity in severityTypes) {
      return severity as Severity;
    } else {
      return 'none';
    }
  }
```